core/modules/user/user.admin.inc

```diff
@@ -399,14 +399,53 @@ function user_login_settings($form, &$form_state) {
     ),
   );
 
+  $form['user_login_page_settings'] = array(
+    '#type' => 'fieldset',
+    '#title' => t('Login page settings'),
+  );
+  $form['user_login_page_settings']['user_login_page_tabs'] = array(
+    '#title' => t('Use tabs'),
+    '#type' => 'checkbox',
+    '#description' => t('If selected shows local tabs for "Create new account", "Log in", and "Reset passord" links. A simplified page with links is shown if unchecked.'),
+    '#default_value' => $core_config->get('user_login_page_tabs'),
+    '#config' => 'system.core',
+  );
+  $form['user_login_page_settings']['user_login_page_suppress_layout'] = array(
+    '#title' => t('Hide layout regions'),
+    '#type' => 'checkbox',
+    '#description' => t('Hide all layout regions, such as headers, footers, and sidebars. Only applies if the "Use tabs" setting is off.'),
```

Comment:
Perhaps do something like this, so that we don't have to keep changing the translation if the label of the setting happens to change:
```suggestion
    '#description' => t('Hide all layout regions, such as headers, footers, and sidebars. Only applies if the %user_login_page_tabs setting is off.' array('%user_login_page_tabs' => $form['user_login_page_settings']['user_login_page_tabs']['#title'])),
```

Reply:
This is more complicated.